src/features/vars-schema/env-vars.ts

```diff
@@ -363,6 +403,14 @@ const checkVarName = (type: string, name: string) => {
 	}
 };
 
+const checkAlphaNumeric = (type: string, name: string) => {
+	if (/^\d|\W/.test(name)) {
```

Comment:
In-lining just this one case feels weird. I'd say either declare it like `INVALID_CHARACTER_REGEX` or in-line both and have their names document the regex purpose